ci-operator/config/openshift/console-plugin-template/openshift-console-plugin-template-main.yaml

```diff
@@ -0,0 +1,55 @@
+base_images:
+  os:
```

Comment:
this will need to be:

```yaml
  nodejs-10:
    name: nodejs-10-rhel7
    namespace: openshift
    tag: latest
```

if we will go with nodejs-10, based on my other [comment](https://github.com/openshift/console-plugin-template/pull/9#issuecomment-1323808871)